packages/server/modules/core/graph/resolvers/apitoken.ts

```diff
@@ -45,20 +45,20 @@ const resolvers = {
       })
 
       return await createPersonalAccessToken(
-        context.userId,
+        context.userId!,
         args.token.name,
-        args.token.scopes,
-        args.token.lifespan
+        args.token.scopes.filter(isValidScope),
```

Comment:
@fabis94 API scopes are included in the AllScopes I suppose

Reply:
@alemagio they are, this prevents from users from submitting nonexistant scopes which may have been possible to do before?